src/test/java/ch/heigvd/amt/mvcProject/application/question/QuestionFacadeTestIT.java

```diff
@@ -238,4 +244,137 @@ public void getQuestionById_ShouldThrownError_IfQuestionIdDoesntExist() {
         });
     }
 
+    @Test
+    public void upvoteQuestionShouldWork() throws UserFailedException, QuestionFailedException {
+        QuestionCommand command = QuestionCommand.builder()
+                .title("Titre")
+                .description("Description")
+                .creationDate(new Date())
+                .userId(currentUserDTO.getUserId())
+                .build();
+
+        QuestionsDTO.QuestionDTO question = questionFacade.addQuestion(command);
+
+        questionFacade.upvote(currentUserDTO.getUserId(), question.getId());
+
+        QuestionsDTO.QuestionDTO upvotedQuestion = questionFacade.getQuestion(QuestionQuery.builder().questionId(question.getId()).build());
+
+        assertEquals(1, upvotedQuestion.getVotes());
+    }
+
+    /*@Test
+    public void upvotingTwiceRemovesTheUpvote() throws UserFailedException, QuestionFailedException {
```

Comment:
Les tests d'intégration devrait marcher. J'insiste pas, mais avant Vendredi, il faut qu'il y ait du vert dans console.
![image](https://user-images.githubusercontent.com/44243798/97110711-0ad03c00-16db-11eb-8cf8-7f9387a1e213.png)
Hormis ça j'ai une erreur dans ton seul test d'intégration

Reply:
[WIP] in title ;_;